cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/ProductsEndpointInput.java

```diff
@@ -0,0 +1,34 @@
+package mil.army.usace.hec.cumulus.client.controllers;
+
+import static mil.army.usace.hec.cumulus.client.controllers.CumulusEndpointConstants.ACCEPT_HEADER_V1;
+import static mil.army.usace.hec.cumulus.client.controllers.CumulusEndpointConstants.ACCEPT_QUERY_HEADER;
+
+import java.util.Objects;
+import mil.army.usace.hec.cwms.http.client.EndpointInput;
+import mil.army.usace.hec.cwms.http.client.HttpRequestBuilder;
+
+public class ProductsEndpointInput extends EndpointInput {
+
+    static final String PRODUCT_ID_PARAMETER = "product_id";
+
+    private final String productId;
+    /**
+     * Endpoint Input for Products to handle input parameters.
+     *
+     * @param productId - ID of product (or null if retrieving all products)
+     */
+
+    public ProductsEndpointInput(String productId) {
+        this.productId = Objects.requireNonNull(productId, "Cannot access acquirable file without acquirable id");
```

Comment:
aquirable file in product endpoint? Does this API download files?

Reply:
That's a typo that I missed. I'll fix it.

Reply:
fixed it